lists/gr.csv

```diff
@@ -38,3 +38,8 @@ https://www.migrant.gr/,HUMR,Human Rights Issues,2018-09-20,OONI,
 http://www.efsyn.gr/,NEWS,News Media,2018-09-20,OONI,
 http://astegoi.gr/,HUMR,Human Rights Issues,2018-09-20,OONI,
 https://www.amnesty.gr/,HUMR,Human Rights Issues,2018-09-20,OONI,
+http://www.subtitles.gr,SRCH,Search Engines,2018-11-09,Anonymous,
+http://www.subztv.club/,CULTR,Culture,2018-11-09,Anonymous,
+http://www.yts.am/,FILE,File-sharing,2018-11-09,Anonymous,
+http://www.loosecannonholdem.com/,MISC,Miscelaneous content,2018-11-09,Anonymous,
```

Comment:
This should probably go under control content (we try to avoid using MISC for all new PRs and it was only added due to some incompatibility in the new vs old category codes).

Reply:
Thanks for the review, updated the category.